bazel/repository_locations.bzl

```diff
@@ -1376,6 +1376,19 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2022-03-30",
         cpe = "N/A",
     ),
+    rules_license = dict(
+        project_name = "rules_license",
+        project_desc = "Bazel rules for checking open source licenses",
+        project_url = "https://github.com/bazelbuild/rules_license",
+        version = "0.0.3",
+        sha256 = "00ccc0df21312c127ac4b12880ab0f9a26c1cff99442dc6c5a331750360de3c3",
+        urls = ["https://github.com/bazelbuild/rules_license/releases/download/0.0.3/rules_license-0.0.3.tar.gz"],
```

Comment:
I didn't see this one come through - we shouldn't hard code the versions here. Should be
`        urls = ["https://github.com/bazelbuild/rules_license/releases/download/{version}/rules_license-{version}.tar.gz"],`

Reply:
oops thanks! https://github.com/envoyproxy/envoy/pull/23780